numpy/polynomial/chebyshev.py

```diff
@@ -1647,6 +1647,129 @@ def chebfit(x, y, deg, rcond=None, full=False, w=None):
     return pu._fit(chebvander, x, y, deg, rcond, full, w)
 
 
+def chebfit2d(x, y, z, deg, rcond=None, full=False, w=None, max_degree=None):
+    """
+    2d Least squares fit of Chebyshev series to data.
+
+    Return the coefficients of a Chebyshev series of degree `deg` that is the
+    least squares fit to the data values `y` given at points `x`.
+    The fitted polynomial(s) are in the form
+
+    .. math::  p(x, y) = c_00 + c_10 * T_10(x, y) + ... + c_nm * T_nm(x, y),
+
+    where `n` and `m` are `deg`.
```

Comment:
Perhaps a good idea to add `..versionadded:: 1.19.0` here (I doubt very much this will make 18)

Reply:
Added